### DISCUSSION TITLE
Comments not included in export from opportunities modules

### DISCUSSION THREAD

**Discussion by testmodem** [YetiForceCompany/YetiForceCRM#16462] [General] [answered]
Dear concern .. 

When I export my records from "Opportunities module" as csv, the export doesn't include "comments" of relevant oppertunities, comments column is empty in csv file.

Please advise.
![comments-not-imcluded](https://user-images.githubusercontent.com/59494950/189745006-be7aa92d-5e71-4fd1-b936-0dfb7fdab9ea.png)

![csv-no-comments](https://user-images.githubusercontent.com/59494950/189746402-80cb9363-1bc6-429f-9dc3-864e19eff4db.png)

**Comment by vovpff** [comment, marked as answer]
This is not for related comments. This is for field named Comments

**Reply by testmodem** [reply]
Thanks for replying . Can you please explain how can i export "related comments ?

![need-this](https://user-images.githubusercontent.com/59494950/189841221-459c7bd6-bfd6-464e-87aa-ab94ce859e2a.png)

**Reply by vovpff** [reply]
I think this is impossible with standard tools. Programmer can write script what export data with related comments as you need. You can do it by yourself or charge to YF dev crew

**Reply by testmodem** [reply]
Thanks ..  noted ..

**Reply by testmodem** [reply]
@vovpff Can you please advise  where does this "Comments" field shows up in opportunity form ? I was unable to locate it .. I think there is a "Comments" widget that shows 'current reocrd' and 'related record'. 

A bit clueless please advise..

**Reply by vovpff** [reply]
I'm sorry but I'm away from my PC.

I will describe as I remember:

There is some table named like `...modcomments`. This table contains whole comments from your CRM. You can find all comments related to opportunity via `related_to` column with filled with opportunity crmid.

**Reply by mariuszkrzaczkowski** [reply]
there is another way to export a comment, there is a comment list option in PDF:
```$(record : Comments 50)$```

**Reply by mariuszkrzaczkowski** [reply]
it also works for email templates

**Reply by testmodem** [reply]
> I'm sorry but I'm away from my PC.
> 
> I will describe as I remember:
> 
> There is some table named like `...modcomments`. This table contains whole comments from your CRM. You can find all comments related to opportunity via `related_to` column with filled with opportunity crmid.

Thank you very much .. I tried and was able to get the data. Is there anyway i can obtain it with its timestamp?
Also is there a easier way to do it .. I hope someone can make a tutorial about it . or at least a way to expect all such data through CSV at least . Beginners like me are really struggling .. 

Thanks

**Reply by testmodem** [reply]
> > I'm sorry but I'm away from my PC.
> > I will describe as I remember:
> > There is some table named like `...modcomments`. This table contains whole comments from your CRM. You can find all comments related to opportunity via `related_to` column with filled with opportunity crmid.
> 
> Thank you very much .. I tried and was able to get the data. Is there anyway i can obtain it with its timestamp? Also is there a easier way to do it .. I hope someone can make a tutorial about it . or at least a way to expect all such data through CSV at least . Beginners like me are really struggling ..
> 
> Thanks

Thank you for your response .. I tried but then i was totally blank .

![image](https://user-images.githubusercontent.com/59494950/190447430-43361f49-7cee-42c9-bc2c-6cbb91f78dc3.png)
